docs/adminguide/hosting/index.rst

```diff
@@ -14,7 +14,7 @@
 Parsec Server installation
 ==========================
 
-This guide covers the installation procedure for the `Parsec`_ server (also called `parsec-backend`).
+This guide covers the installation procedure for the `Parsec`_ server (also called `parsec-server`).
```

Comment:
The "also called" seems weird because it's just a case kebab-case (and `parsec-server` already makes you think about Parsec server)  
```suggestion
This guide covers the installation procedure for the `Parsec`_ server.
```